src/main/java/io/vlingo/http/resource/ErrorHandler.java

```diff
@@ -1,12 +1,14 @@
 package io.vlingo.http.resource;
 
+import io.vlingo.actors.Logger;
 import io.vlingo.common.Completes;
 import io.vlingo.http.Response;
 
 public interface ErrorHandler {
-  Completes<Response> handle(Throwable error);
 
-  static ErrorHandler handleAllWith(Response.Status status) {
-    return error -> Completes.withSuccess(Response.of(status));
+  Completes<Response> handle(final Throwable error, final Logger logger);
```

Comment:
My only comment is on this, why `handle` method requires the Logger?

Reply:
That was my reflection as well, which I shared in the gitter channel and above.  Please see my follow-up commit.  Thanks for reviewing!